pkg/log/ansHook.go

```diff
@@ -0,0 +1,112 @@
+package log
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/SAP/jenkins-library/pkg/ans"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"io/ioutil"
+	"strings"
+)
+
+// ANSHook is used to set the hook features for the logrus hook
+type ANSHook struct {
+	client ans.Client
+	event  ans.Event
```

Comment:
Yes, it's for defaults. I changed the name to `defaultEvent`

Reply:
shouldn't it be eventDefaults?

Reply:
I thought about it and decided against that, because this is a mix of two things: 
- the eventDefaults, as defined by us
- the users event template, as defined by the user

My thinking is that the result of this mix is more a defaultEvent, rather than eventDefaults. But we could also say that the user defined his own event defaults with their template and added it to the mix. Either or is fine by me, what do you think @rodibrin ?

Reply:
The variable name `event` or `defaultEvent` as well as the variable type `ans.Event` point to an event, which isnt true. Instead, its a default settings collection, a Map actually.

Reply:
yes, template makes more sense.

Reply:
done